resources/prometheus/prometheus-rules.yaml

```diff
@@ -538,104 +538,95 @@ spec:
         - record: strictly_worker_nodes
           expr: |
             kube_node_role{role="worker"} * on (node) (sum(kube_node_role) by (node) == 1)
-        - record: node_availability_zone
```

Comment:
We currently show the AZ resources in the dashboard as well. I would keep the recording rules and just change the alerts.

Reply:
Done

Reply:
Eh I meant in the other recording rules as well. So that in the end we have both `memory_resource_requests:strictly_worker_nodes:sum` and `memory_resource_requests:strictly_worker_nodes:by_availability_zone:sum`. The latter are still useful to gauge the distribution between the AZs.

Reply:
Ok, done.